.pre-commit-config.yaml

```diff
@@ -16,8 +16,12 @@ repos:
           - javascript
           - css
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.1.2
+    rev: v0.4.1
     hooks:
       - id: ruff-format
+        exclude: "^docs/(tutorials|examples)/gallery_.*?\\.py$"
+      - id: ruff-format
+        files: "^docs/(tutorials|examples)/gallery_.*?\\.py$"
+        args: [--line-length=83]
```

Comment:
The galleries use a smaller maximum line length to avoid vertical scrollbars. Example in the comment below.